src/oop/_OOP_find_proc_info.c

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
nit: use the libunwind header.

Reply:
+1 Yea if possible change to libunwind header.